xknxproject/zip/extractor.py

```diff
@@ -167,6 +171,11 @@ def _get_schema_version(namespace: str) -> int:
     return schema_version
 
 
+def _is_ets4_project(schema_version: int) -> bool:
+    """Check if the project is an ETS4 project."""
+    return schema_version == ETS_4_SCHEMA_VERSION
```

Comment:
```suggestion
    return schema_version <= ETS_4_2_SCHEMA_VERSION
```
Afaic 11 stands for `1.1` - so 10 would be `1.0` which is probably the lowest version we will see.
Even if there were changes from 10 to 11 we don't know yet, I guess we can at least try to parse it for now...

Reply:
Sounds good.

Reply:
Done